.github/workflows/release-operator-container.yml

```diff
@@ -0,0 +1,63 @@
+name: Release Keycloak Operator container
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+concurrency: keycloak-operator-container-release
+
+env:
+  JDK_VERSION: 11
+
+jobs:
+  publish:
+    if: github.repository == 'keycloak/keycloak'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Container metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: quay.io/keycloak/keycloak-operator
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JDK_VERSION }}
+
+      - name: Cache Maven packages
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+          key: cache-1-${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: cache-1-${{ runner.os }}-m2
+
+      - name: Build locally Keycloak artifacts to avoid depending on Maven
+        run: |
+          mvn clean install -DskipTests -DskipExamples -DskipTestsuite
```

Comment:
IIRC, we actually wanted to depend on Maven central so we could build just the operator module in this workflow. Or not? @stianst

Reply:
From experience, the sync with Maven Central can take several hours (even 1 day or so), relying on the local cache prevents to be affected from those delays.
I warmly suggest we use the local cache, but if you understand the downsides and still prefer to rely on upstream published jars I'm ready to adapt this PR.

Reply:
I don't have a strong opinion about this. Let's wait @stianst.

Reply:
Would prefer to just pull the actually released artefacts rather than building here. We could pull those directly from https://s01.oss.sonatype.org/ instead of Maven Central as they will be available there straight away. 

Since this is just a temporary solution until we move to the new release process I'm okay with this for now. However, building the whole of Keycloak just because you need keycloak-common is quite pointless. 

If you build the Operator with:

`mvn -P operator -pl operator -am clean package`

That should take care of building the Operator and the dependencies it needs.

Reply:
Updated the command to:
```bash
mvn -P operator -pl operator -am clean install -DskipTests
```
it's indeed much faster! Thanks for the suggestion!